### PR TITLE
libwebsockets: Add fuzzer for lws-upng

### PR DESCRIPTION
**Pull request by autofuzzoss**
This fuzzer is to fuzz lws-upng API function in libwebsockets project.
I recommended the fuzzer to maintainer of libwebsockets in https://github.com/warmcat/libwebsockets/issues/2687

I suggests to add this fuzzer and environment files for test for continuous fuzz testing.

### CODE REVIEW

File: projects/libwebsockets/project.yaml [hunk 1]
```diff
@@ -3,7 +3,6 @@ language: c
 primary_contact: "andy@warmcat.com"
 sanitizers:
   - address
-  - memory:
```
Comment (jonathanmetzman) [inline projects/libwebsockets/project.yaml:6]: You don't want MSAN and UBSAN?

**Review by jonathanmetzman — APPROVED** [review]
LGTM

File: projects/libwebsockets/project.yaml (hunk 1 above)
Comment (autofuzzoss) [reply projects/libwebsockets/project.yaml:6]: I disabled to avoid build fail.

Comment (jonathanmetzman) [reply projects/libwebsockets/project.yaml:6]: probably better to fix if possible (UBSAN will probably be a lot easier to fix than MSAN which will require compiling everything linked against your binary with MSAN)

**Pull request merged**